canonical-web-and-design/canonical.com
# Port the projects section from the old site

## Done
Copied over the content from the old projects section of the site as its a legal requirement.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Go to the footer and click Projects
- Check the page loads and looks ok
- Click on the directory link under the main projects
- Check the directory page renders
